src/utils/jsHelper/helper.tsx

```
@@ -276,3 +276,50 @@ export const formatDate = (dateString) => {
+    // Convert camelCase to space-separated
+    const spacedKey = key.replace(/([a-z])([A-Z])/g, "$1 $2");
+
+    // Convert snake_case to space-separated
+    const normalizedKey = spacedKey.replace(/_/g, " ");
+
+    // Capitalize each word
+    return normalizedKey.replace(/\b\w/g, (char) => char.toUpperCase());
+  }
+
+  for (const key in applicationData) {
+    if (applicationData.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)